### ISSUE TITLE
Table that supports the Platform API needs a 'usage' field periodicaly updated

### ISSUE THREAD

**Issue by pmorrill** [MotusDev/Motus-TO-DO#193]
The table: adm_platform_api
The field: usageValue (int)

We would like to update this field using a scheduled script of some sort, probably populating it with the current tag-detections count.

We also need a 'last reset' date in the same table (datetime field is 'lastReset'). This date can be the earliest valid tag detection time that we have.

**Comment by denislepage** [comment]
I am not seeing a field for when the stat was generated, or did I miss it? I would have assumed that last reset was the datetime when you calculate the stats, not the starting point, but the documentation seems clear on that:
https://www.canarie.ca/wp-content/uploads/Research-Platform-Support-for-the-CANARIE-Registry-and-Monitoring-System.pdf

I have used the earliest batch with data, excluding errors due to gps readings. That gives us ‘2013-03-20 19:56:29.727’ as a starting point. This predates the platform itself, so I may need to confirm they are ok with our metric.

select dbo.ts2datetime(MIN(tsBegin)) from sg_batches
where tsBegin >= dbo.datetime2ts('2010-01-01') and numHits > 0

I have now created a proc that will update the adm_platform_api table. It’s very quick, so we could update it regularly, at least a few times a day. I’ll add this to the schedule.

I am also updating a stats table called adm_platform_stats, which has the following names (and values):

``
stat_name          stat_value
projects               178
recv       832
recv_depl            2183
sg_taghits           433274155
tags_depl            13206
users_all              611
users_pi               131
``

It’d be nice to display those on our home page. I am wondering if the CANARIE API entry point allows for multiple values. I can ask them. Or we could just create a distinct entry point that we could use on the home page to build a widget. We would likely need to give human-readable values for stat_name.

I have also changed the support email to motus@birdscanada.org<mailto:motus@birdscanada.org>
